src/Swarm/TUI/Model.hs

```diff
@@ -258,6 +265,15 @@ uiFPS :: Lens' UIState Double
 -- | The base-2 logarithm of the current game speed in ticks per
 --   second.
 lgTicksPerSecond :: Lens' UIState Int
+lgTicksPerSecond = lens _lgTicksPerSecond safeSetLgTicks
+ where
+  maxTicks = 5 -- 2^(-5) == 1/32
+  minTicks = 1 - floor (logBase 2 (fromIntegral (maxBound :: Int)) :: Float)
```

Comment:
This seems unnecessarily complicated.  Why not just have it be something arbitrary like -30 and be done with it?

Reply:
I do not know *what value* makes sense, so I left it both in (one less then) maximum. If the players want to skip ahead from the bronze age to Rome or just [don't want to use the pause](https://xkcd.com/1172/), I will not stop them. :smirk: